rust/src/core.rs

```diff
@@ -8,6 +8,12 @@ use crate::ffiutil;
 use ffiutil::*;
 use openat_ext::OpenatDirExt;
 
+/// The binary forked from useradd that pokes the sss cache.
+/// It spews warnings (and sometimes fatal errors) when used
+/// in a non-systemd container (default treecompose side) so
+/// we replace it with `true`.
```

Comment:
Filed https://github.com/SSSD/sssd/issues/5687! Let's link to that somewhere around here?

Reply:
Our CI is slow and cumbersome enough that I find myself hesitating to re-push for things like tweaking comments.  I know this debatable but for "minor nits" maybe we can establish a flow where we merge as is for things like this and do followup PRs?

Reply:
Sure, WFM.

Reply:
(needs review/approve then if you're good!)

Reply:
As promised fixup in https://github.com/coreos/rpm-ostree/pull/2906 along with other bits